jwst/datamodels/schemas/dark.schema.yaml

```diff
@@ -1,25 +1,30 @@
 title: Dark current data model
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: instrument_name.schema.yaml
+- $ref: detector.schema.yaml
+- $ref: filter_core.schema.yaml
+- $ref: exposure_type.schema.yaml
+- $ref: subarray.schema.yaml
 - type: object
   properties:
     data:
       title: Dark current array
       fits_hdu: SCI
       default: 0.0
-      ndim: 3
+      ndim: 4
       datatype: float32
     dq:
       title: 2-D data quality array for all planes
       fits_hdu: DQ
       default: 0
-      ndim: 2
-      datatype: uint32
+      ndim: 4
+      datatype: int16
     err:
       title: Error array
       fits_hdu: ERR
       default: 0.0
-      ndim: 3
+      ndim: 4
```

Comment:
Ah, good catch. I was testing a fix to a memory problem in asdf and forgot to reset this. Will push the fix with the next update.